src/parsers/maven_parser.py

```diff
@@ -42,7 +42,9 @@ def _parse_string(coordinates_str):
         elif ncolons == 3:
             a['groupId'], a['artifactId'], a['packaging'], a['version'] = coordinates_str.split(':')
         elif ncolons == 4:
-            a['groupId'], a['artifactId'], a['packaging'], a['classifier'], a['version'] = \
+            # Usually, it's groupId:artifactId:packaging:classifier:version but here it's
```

Comment:
@abs51295 - will it work with output of other maven plugins? If not, then we should think of a more generalized solution.